package.json

```diff
@@ -25,25 +25,26 @@
     "eslint-config-react-app": "^3.0.8",
     "eslint-plugin-class-property": "^1.1.0",
     "eslint-plugin-flowtype": "^3.13.0",
-    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-import": "^2.29.0",
     "eslint-plugin-jsdoc": "^29.2.0",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
-    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
+    "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-sort-destructure-keys": "^1.3.5",
+    "eslint-plugin-sort-destructure-keys": "^1.5.0",
     "jest": "^24.8.0",
     "jest-cli": "^24.8.0",
     "prop-types": "^15.6.2",
     "react": "^16.4.2",
     "react-dom": "^16.4.2",
     "react-select": "^4.3.1",
     "react-test-renderer": "^16.4.2",
-    "sass": "^1.44.0",
-    "webpack": "^4.44.2",
+    "sass": "^1.69.5",
+    "webpack": "^4.47.0",
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0"
   },
   "dependencies": {
+    "dompurify": "^3.0.6",
```

Comment:
I'm sorry, @MiguelAxcar , I don't know how I missed that you had already installed this 🤦 🤔 

I also now understand that the errors we're seeing are because I steered you wrong: DOMPurify wants to be imported like this,

```js
import * as DOMPurify from 'dompurify';
```
so then we _would_ call it as `DOMPurify.sanitize()` in the way you did before.

Best way to get things working is probably to revert 7d95e8c658d4792dd9e52b28faa9a925434f3ad8 and then change the imports to match the format above.

Apologies for the off-target guidance!